runtime/build.rs

```diff
@@ -1,13 +1,5 @@
-#[cfg(feature = "std")]
+use substrate_wasm_builder::WasmBuilder;
+
 fn main() {
-    substrate_wasm_builder::WasmBuilder::new()
-        .with_current_project()
-        .export_heap_base()
-        .import_memory()
-        .build()
+    WasmBuilder::new().with_current_project().export_heap_base().import_memory().build()
 }
-
-/// The wasm builder is deactivated when compiling
-/// this crate for wasm to speed up the compilation.
-#[cfg(not(feature = "std"))]
-fn main() {}
```

Comment:
can you inform us on why this part is removed, and why it was not necessary?

Reply:
https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/parachain-template/runtime/build.rs 

in case there was a misunderstanding, we didn't make any changes. This is from substrate